### PR TITLE
Added Wiki DataModule

### PR DESCRIPTION
**Pull request by cicheck**
# What?
* Added wiki DataModule from original code.
* As you can see it is splitted into three tasks: aggression, attack and toxicity

### CODE REVIEW

File: personalized_active_learning/experiments/active_learning/wiki/aggression.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        }
+    )
+    model_kwargs_list = product_kwargs({})
+    trainer_kwargs_list = product_kwargs(
+        {
+            "epochs": [20],
+            "lr": [0.0001],
+            "use_cuda": [use_cuda],
+            "monitor_metric": ["valid_loss"],
+            "monitor_mode": ["min"],
```
Comment (kaszycaO) [inline personalized_active_learning/experiments/active_learning/wiki/aggression.py:81]: [Question] Is it the best configuration of params? Or it's just copy paste? (random)

**Comment by kaszycaO** [comment]
:+1:

File: personalized_active_learning/experiments/active_learning/wiki/aggression.py (hunk 1 above)
Comment (cicheck) [reply personalized_active_learning/experiments/active_learning/wiki/aggression.py:81]: The second:P

Comment (cicheck) [reply personalized_active_learning/experiments/active_learning/wiki/aggression.py:81]: I.e. copy paste

**Pull request merged**